Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:lts-alpine
+
+# drop back to the non-privileged user for run-time
+WORKDIR /home/node
+USER node
+
+# tell node, et al to run in production mode
+ENV NODE_ENV production
+
+# copy in the package files so that we can install and build the project
+# dependencies
+COPY --chown=node:node package*.json ./
+
+# install all the node modules required
+RUN npm ci
+
+# copy the code from the project
+COPY --chown=node:node ./src ./src
+
+# these variables are for overriding but keep them consistent between image and
+# run
+ENV FC_API_PORT 3005
```

Comment:
Is this right? Normally it's 1 higher than the FE app, and I think FaC is running on 3008.

Reply:
Resolved in f814f43a997e11851bbfff4a3942b4c3f17a6856.